pocean/utils.py

```diff
@@ -264,6 +264,8 @@ def get_fill_value(var):
 
 def create_ncvar_from_series(ncd, var_name, dimensions, series, **kwargs):
     from pocean.cf import CFDataset
+    kwargs["zlib"] = kwargs.get("zlib", True)
+    kwargs["complevel"] = kwargs.get("complevel", True)
```

Comment:
This needs to be `1`

Reply:
Ooops. Copy-n-pasta. Fixed in the next commit.